tests/utils/stl/test/format.py

```diff
@@ -121,7 +121,7 @@ def setup(self, test):
         # on the .dat files the test requires.
         for path in source_dir.iterdir():
             if path.is_file() and path.name.endswith('.dat'):
-                os.link(path, exec_dir / path.name)
+                shutil.copy2(path, exec_dir / path.name)

```

Comment:
Why is this change necessary? The PR description doesn't say.

Reply:
You can't create a hard-link across drives, so I just copy the tiny .dat files around instead.